api/request_handler.go

```diff
@@ -36,6 +39,17 @@ var upgrader = websocket.Upgrader{
 func (s *RequestHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	v := r.URL.Query()
 	pitayaAddress := v.Get("address")
+	upgrader.CheckOrigin = func(r *http.Request) bool {
```

Comment:
This global upgrader is weird, but it seems to be the way the gorilla developers want it to be used =/